app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
@martinpovolny then can we make this one-line file from a HAML file to a standard HTML file with `<!DOCTYPE html>`?